layers/+window-management/exwm/packages.el

```diff
@@ -29,42 +29,33 @@
   (use-package xelb))
 
 (defun exwm/init-exwm ()
+  (use-package exwm-randr)
+  (use-package exwm-systemtray)
   (use-package exwm
     :init
     ;; Disable dialog boxes since they are unusable in EXWM
     (setq use-dialog-box nil)
-    ;; 10 Worskpaces please
-    (setq exwm-workspace-number 10)
+    ;; Use as many workspaces as there are displays
+    (setq exwm-workspace-number 3)
     ;; You may want Emacs to show you the time
     (display-time-mode t)
     (when exwm--hide-tiling-modeline
       (add-hook 'exwm-mode-hook #'hidden-mode-line-mode))
-    ;; Trying to make shell-pop with a real terminal :P
-    ;; (defun exwm-launch-term ()
-    ;;   (start-process-shell-command exwm--terminal-command
-    ;;                                nil exwm--terminal-command))
-    ;; (defun shell-pop-exwm-term (index)
-    ;;   (interactive "P")
-    ;;   (require 'shell-pop)
-    ;;   (shell-pop--set-shell-type
-    ;;    'shell-pop-shell-type
-    ;;    '("exwm-term"
-    ;;      "Termite" #'exwm-launch-term))
-    ;;   (shell-pop index))
-    :config
-    (when dotspacemacs-use-ido
-      (exwm-enable-ido-workaround))
```

Comment:
I'm not sure why we wouldn't want to allow `ido` users to use ido.

Reply:
This was deprecated, I've added the current support for ido in the next set of commits.